apps/src/music/views/SoundsPanel.jsx

```diff
@@ -42,6 +28,7 @@ const SoundsPanelRow = ({
   const soundPath = folder.path + '/' + sound.src;
   const isSelected = soundPath === currentValue;
   const isPlayingPreview = playingPreview === soundPath;
+  const typeIconPath = `/blockly/media/music/icon-${sound.type}.png`;
```

Comment:
nit: might be good to pull this out into a shared helper function that converts `SoundType` to url, to keep it consistent with FieldSounds.js?